omero/sysadmins/version-requirements.rst

```diff
@@ -45,34 +37,42 @@ Operating systems
      :header-rows: 1
 
      * - Distribution version
-       - OMERO 5.3
        - OMERO 5.4
        - OMERO 5.5
+       - OMERO 5.6
      * - 6.x
-       - |Deprecated|
        - |Deprecated|
        - |Dropped|
+       - |Dropped|
      * - 7.x
        - |Recommended|
        - |Recommended|
        - |Recommended|
+     * - 8.x
+       - |Upcoming|
+       - |Upcoming|
+       - |Upcoming|
```

Comment:
Not `Recommended` for the next version? If it's tentative, it seems like we should at least tentatively suggest what the recommendation will be.